### PR TITLE
fix(element): make .count() work correctly on NoSuchElement exception

### PR DESCRIPTION
**Pull request by jafd**
Unfortunately, 88dd568c5295234a5211a23e12666e199606e437 does not fix the issue #3079, but swaps one error with another.

The problem is that comparing integers to objects doesn't work, so the problem persists. We need to compare ```code``` to the corresponding ```code```, and this is what my patch does. It has been tested to work with tests of the project I'm working on; they were failing in 3.2.x and 4.0.0.

### CODE REVIEW

**Comment by jafd** [comment]
Please verify; I have signed the CLA.

**Comment by jafd** [comment]
Is it okay now?

File: lib/element.ts [hunk 1]
```diff
@@ -392,7 +392,7 @@ export class ElementArrayFinder extends WebdriverWebElement {
     return this.getWebElements().then(
         (arr: any) => { return arr.length; },
         (err: any) => {
-          if (err.code == new webdriver.error.NoSuchElementError()) {
+          if (err.code == webdriver.error.NoSuchElementError.code) {
```
Comment (Droogans) [inline lib/element.ts]: This can be `===` now that they types match.

**Comment by jafd** [comment]
Will have to re-make this one...

**Pull request closed without merging**